src/main/java/net/montoyo/mcef/client/ClientProxy.java

```diff
@@ -320,6 +326,8 @@ public void onShutdown() {
             Thread.sleep(100);
         } catch(Throwable t) {}
 
+        // DOM: TODO: MCEF: we may need to call this from the other thread, getting this failed check and crash:
+        // [0617/001244.151:FATAL:context.cpp(129)] Check failed: thread_checker_.CalledOnValidThread()
         cefApp.N_Shutdown();
```

Comment:
this was crashing the debug DLL which was failing an assert complaining about us calling Shutdown() on the wrong thread.  probably should check this out, unless you think ShutdownPatcher is supposed to hack around this.

I think we'll need to add in the original JCEF code here which you can see a similar example at: https://github.com/binary1230/mcef/commit/3d26ca58dbe867a832cf2edf26d5a47cf0310359